### PR TITLE
fix yume,_inc name

### PR DESCRIPTION
**Pull request by jellizaveta**

### CODE REVIEW

File: dist/trackers.json [hunk 1]
```diff
@@ -19877,7 +19877,7 @@
             "url": "http://www.yume.com/",
             "companyId": "yume"
         },
-        "yume,_inc.": {
+        "yume_inc": {
```
Comment (ameshkov) [inline dist/trackers.json:19880]: Actually, it would be better to fix the script that generates the CSV, not the data (since it came from whotracksme).

Comment (jellizaveta) [reply dist/trackers.json:19880]: Add escaping at build for such titles or exclude characters that are worth escaping?

Comment (ameshkov) [reply dist/trackers.json:19880]: Better add escaping

File: index.ts [hunk 2]
```diff
@@ -270,6 +270,18 @@ function buildTrackersDomains(
     return sortRecordsAlphabetically(merged, false);
 }
 
+/**
+ * Escape special characters in a string for CSV format.
+ * @param {string} value - The input string to escape.
+ * @returns {string} - The escaped string.
+ */
+function escapeCSV(value: string): string {
+    if (value.includes(',') || value.includes('"')) {
```
Comment (ameshkov) [inline index.ts:279]: Давай просто вот такой библиотекой CSV формировать:
https://www.npmjs.com/package/csv-stringify

И вкоммить еще dist измененный для проверки

**Review by ameshkov — APPROVED** [review]

**Pull request merged**